pkg/reconciler/revision/background.go

```diff
@@ -206,6 +218,20 @@ func (r *backgroundResolver) addWorkItems(rev *v1.Revision, name types.Namespace
 		r.results[name].workItems[i] = item
 		r.queue.AddRateLimited(item)
 	}
+
+	for i, container := range rev.Spec.InitContainers {
+		item := workItem{
+			revision:        name,
+			timeout:         timeout,
+			name:            container.Name,
+			image:           container.Image,
+			index:           i,
+			isInitContainer: true,
+		}
+
+		r.results[name].workItems[len(rev.Spec.Containers)+i] = item
```

Comment:
might be better to use `append` here (and do `make([]workItem, 0, totalNumOfContainers)` above)

Reply:
ok let me check